### DISCUSSION TITLE
I need to handle the event of selecting a row, but could not find it in the docs

### DISCUSSION THREAD

**Discussion by andreidiaconescu** [KevinVandy/mantine-react-table#379] [Q&A]
In the docks they only talk about Row Selection State but it does not seem to help me to know which row was just selected / unselected.
Any idea please ?

**Comment by KevinVandy** [comment]
mantineSelectCheckboxProps

**Reply by andreidiaconescu** [reply]
thank you for the answer,
mantineSelectCheckboxProps is not documented; i looked directly in the typesript types, but ... it could understand how to use it;

in the end end i used selection state, even if it seems more work;
